src/relay/components/Setting.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+
+class Setting extends React.Component<SettingProp> {
```

Comment:
I strongly recommend using functional components instead of class components. The difference is really night and day.

Reply:
Done. As the components get more complex, do you switch to class components?

Reply:
Nope. Those are really the old way of writing components - the new way is to use hooks.